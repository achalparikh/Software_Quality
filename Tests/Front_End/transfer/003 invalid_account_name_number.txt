login
standard
ACHAL PARIKH
transfer
00002
logout
quit