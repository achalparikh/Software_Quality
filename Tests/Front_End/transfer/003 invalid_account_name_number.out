Enter user type - "standard" or "admin":
Logging in as standard user - enter name:
You are now logged in as "ACHAL PARIKH"
COMMAND: transfer - enter FROM account number:
ERROR: Account number "00002" does not exist. Try again
You are now logged out
